pkg/gatherers/common/gather_logs.go

```diff
@@ -17,25 +17,32 @@ import (
 	"github.com/openshift/insights-operator/pkg/utils/marshal"
 )
 
-// LogContainersFilter allows you to filter containers
-type LogContainersFilter struct {
-	Namespace                string
-	LabelSelector            string
-	FieldSelector            string
-	ContainerNameRegexFilter string
-	MaxNamespaceContainers   int
+// LogResourceFilter allows you to filter containers
+type LogResourceFilter struct {
+	Namespace                string `json:"namespace"`
+	LabelSelector            string `json:"label_selector,omitempty"`
+	FieldSelector            string `json:"field_selector,omitempty"`
+	ContainerNameRegexFilter string `json:"container_name_regex_filter,omitempty"`
+	PodNameRegexFilter       string `json:"pod_name_regex_filter,omitempty"`
+	MaxNamespaceContainers   int    `json:"max_namespace_containers,omitempty"`
 }
 
 // LogMessagesFilter allows you to filter messages
 type LogMessagesFilter struct {
-	MessagesToSearch []string
-	IsRegexSearch    bool
-	SinceSeconds     int64
-	LimitBytes       int64
-	TailLines        int64
-	Previous         bool
+	MessagesToSearch []string `json:"messages_to_search,omitempty"`
+	IsRegexSearch    bool     `json:"is_regex_search,omitempty"`
+	SinceSeconds     int64    `json:"since_seconds,omitempty"`
+	LimitBytes       int64    `json:"limit_bytes,omitempty"`
+	TailLines        int64    `json:"tail_lines,omitempty"`
```

Comment:
Does it makes sense to still have `SinceSeconds` and `TailLines` parameters here?

Reply:
This is for compatibility with the existing gatherers, if we will not use the conditions we can remove it by limiting the schema.

Reply:
I'll track this as part of the Epic. I think that this is something that we should only remove from the scheme, not from the code.